mozilla/hubs
# Troika Text node

This is the Hubs PR for displaying Troika Text nodes as defined in https://github.com/mozilla/Spoke/pull/1195, based on the troika-three-text package. This package supports a wide range of styling options and has the advantage of being able to display 3D text consistently in Spoke (pure three.js) and Hubs (A-Frame), making it a superior choice for content management purposes versus A-Frame’s <a-text> nodes or Mozilla’s hubs-text.js method.

To test:

1. Follow testing directions in the Spoke PR: https://github.com/mozilla/Spoke/pull/1195
2. Pull and checkout `troika-text` branch of Hubs
3. Run `npm install` or `npm ci` to install the new package
4. Open the scene and check text settings have transferred and display correctly

![image](https://user-images.githubusercontent.com/11968217/143093039-be55e947-41db-48b8-8791-3875754aa5cb.png)
